rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/40_range.yml

```diff
@@ -22,15 +24,40 @@ setup:
         refresh: true
         body:
          - {"index": {}}
-         - { "double" : 42.1, "long": 25 }
+         - { "double" : 42.1, "long": 25, "float": 0.01 }
          - {"index": {}}
-         - { "double" : 100.7, "long": 80 }
+         - { "double" : 100.7, "long": 80, "float": 0.03 }
          - {"index": {}}
-         - { "double" : 50.5, "long":  75}
+         - { "double" : 50.5, "long":  75, "float": 0.04 }
 # For testing missing values
          - {"index": {}}
          - {}
 
+---
+# Regression test for 77033
+"Float Endpoint Exclusive":
+  - do:
+      search:
+        body:
+          size: 0
+          aggs:
+            double_range:
+              range:
+                field: "float"
+                ranges:
+                  -
+                    from: 0
+                    to: 0.04
+                  - from: 0.04
+                    to: 1.0
+  - match: { hits.total.relation: "eq" }
+  - match: { hits.total.value: 4 }
+  - length: { aggregations.double_range.buckets: 2 }
+  - match: { aggregations.double_range.buckets.0.key: "0.0-0.04" }
+  - match: { aggregations.double_range.buckets.0.doc_count: 2 }
+  - match: { aggregations.double_range.buckets.1.key: "0.04-1.0" }
+  - match: { aggregations.double_range.buckets.1.doc_count: 1 }
```

Comment:
Maybe also add an example with `gte` and `gt`. I can never remember if `from` and `to` are inclusive or exclusive. But it's nice to have an example of both.

Reply:
AFAIK the range aggregation doesn't support `gte` and `gt`, more's the pity.

Reply:
I looked it up and, nope. You are right. We don't! We support it for range fields and range queries. Wonderful. Ignore this.